api/src/utils/get-service.ts

```diff
@@ -30,6 +31,10 @@ export function getService(collection: string, opts: AbstractServiceOptions): It
 	switch (collection) {
 		case 'directus_activity':
 			return new ActivityService(opts);
+		// case 'directus_collections':
+		// 	return new CollectionsService(opts);
```

Comment:
Is this an intentional change?

Reply:
```suggestion
		 case 'directus_collections':
		 	return new CollectionsService(opts);
```

Reply:
Circling back here, it seems this was removed due to typing issues and not extending ItemsService. Seems we do not include any services that do not do so.

Removed again in 84311a846cd73321f9a3a695f3f12f49fd33f57c